appveyor.yml

```diff
@@ -70,13 +75,13 @@ environment:
 #      CYGWIN_PYTHON_DEVEL: python27-devel
 
 # TODO: Disabled because tox does nothing in this env.
-#    - TOX_ENV: cygwin32_py36
-#      UNIX_PATH: C:\cygwin\bin
-#      PYTHON_CMD: python3.6
-#      PIP_CMD: pip
-#      CYGWIN_PYTHON_DEVEL: python36-devel
-#      LC_ALL: C.UTF-8
-#      LANG: C.UTF-8
+    - TOX_ENV: cygwin32_py36
```

Comment:
Why does line 77 have the TODO disabled comment still. There is one more of thes further down

Reply:
Because this environment was temporarily enabled for testing the fix. The second commit will be removed again before merging.